application/views/signup_form.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Jeldor
+ * Date: 1/26/15
+ * Time: 19:11
+ */
+?>
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title>新用户注册</title>
+</head>
+<body>
+
+<?php echo validation_errors(); ?>
+
+<?php
+
+echo form_open(site_url('user/signup'));
+echo '学校' . form_input('school', set_value('school')) . br();
+echo '车协名称' . form_input('association_name', set_value('association_name')) . br();
+$provinces = array(
```

Comment:
Another question: temporarily I am storing the province information as strings instead of ids, and I think it is ok to do so. What's your opinion @huxuan ?